modules/apps/captcha/captcha-rest-test/src/testIntegration/java/com/liferay/captcha/rest/resource/v1_0/test/CaptchaResourceTest.java

```diff
@@ -34,7 +36,13 @@ public class CaptchaResourceTest extends BaseCaptchaResourceTestCase {
 	public void setUp() throws Exception {
 		super.setUp();
 
-		CaptchaResource.Builder builder = CaptchaResource.builder();
+		Company company = _companyLocalService.getCompany(
+			TestPropsValues.getCompanyId());
+
+		CaptchaResource.Builder builder = CaptchaResource.builder(
+		).endpoint(
+			company.getVirtualHostname(), 8080, "http"
+		);
```

Comment:
Hey @cdbm, as it is a builder, just do this:

```
_captchaResource = CaptchaResource.builder(
		).endpoint(
			company.getVirtualHostname(), 8080, "http"
		).build();
``` 

and avoid the temp variable